commands/addContributor.ts

```diff
@@ -1,15 +1,44 @@
 import { SlashCommandBuilder } from '@discordjs/builders';
 import { CommandInteraction } from 'discord.js';
-import { isContributor } from '../utils/airTableCalls';
+import { isContributor, createContributor } from '../utils';
+import { createTwitterHandle } from '../utils/twitterHandle';
 
 export const data = new SlashCommandBuilder()
-    .setName('add-contributor')
-    .setDescription('add yourself to the contributor list')
+  .setName('add-contributor')
+  .setDescription('add yourself to the contributor list')
+  .addStringOption(
+    option => option.setRequired(false)
+      .setName('devdao-id')
+      .setDescription('Enter your DevDAO ID (e.g. #2468)'))
+  .addStringOption(
+    option => option.setRequired(false)
+      .setName('twitter')
+      .setDescription('Enter your twitter handle'))
+  .addStringOption(
+    option => option.setRequired(false)
+      .setName('eth-wallet-address')
+      .setDescription('Enter your ETH wallet address'));
 
```

Comment:
I had thought about this when we first made the bot, and I'd like to push back on adding this until we can spend cycles on validating that the person owns the wallet by signing a message. I think eventually we'll be using this table as a source of truth for airdrops, and I would hate for people to have mis-input something here, or not have the right wallet address.

Am I overthinking this? Just want to make sure we get this part right.

Reply:
Maybe I was under thinking it :) I just wanted to include anything that was in the table and the forms. Happy to remove this one for now as I agree with your thinking.